
	<!DOCTYPE html>
	<html lang="zxx" class="no-js">
	
	<script language="javascript" type="text/javascript">

 var state = 'none';
 function showhide(layer_ref) {
 if (state == 'block') {
 state = 'none';
 }
 else {
 state = 'block';
 }

 if (document.all) { //IS IE 4 or 5 (or 6 beta)
 eval( "document.all." + layer_ref + ".style.display = state");
 }
 if (document.layers) { //IS NETSCAPE 4 or below
 document.layers[layer_ref].display = state;
 }
 if (document.getElementById &&!document.all) {
 hza = document.getElementById(layer_ref);
 hza.style.display = state;
 }
 }

 function show(layer_ref) {
 state = 'block';

 if (document.all) { //IS IE 4 or 5 (or 6 beta)
 eval( "document.all." + layer_ref + ".style.display = state");
 }
 if (document.layers) { //IS NETSCAPE 4 or below
 document.layers[layer_ref].display = state;
 }
 if (document.getElementById &&!document.all) {
 hza = document.getElementById(layer_ref);
 hza.style.display = state;
 }
 }

 function hide(layer_ref) {
 state = 'none';

 if (document.all) { //IS IE 4 or 5 (or 6 beta)
 eval( "document.all." + layer_ref + ".style.display = state");
 }
 if (document.layers) { //IS NETSCAPE 4 or below
 document.layers[layer_ref].display = state;
 }
 if (document.getElementById &&!document.all) {
 hza = document.getElementById(layer_ref);
 hza.style.display = state;
 }
 }


 function sDiv(pass) {
 var divs = document.getElementsByTagName('div');
 for(i=0;i<divs.length;i++)
 {
 	if(divs[i].id.match('tag'))
 	{

 	if(divs[i].id.match(pass))
 	{
 		show(divs[i].id);
 	}
 	if(!divs[i].id.match(pass))
 	{
 		hide(divs[i].id);
 	}


 	}
 }

 }

</script>
	
	<head>
		<!-- Mobile Specific Meta -->
		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
		<!-- Favicon-->
		<link rel="shortcut icon" href="img/fav.png">
		<!-- Author Meta -->
		<meta name="author" content="colorlib">
		<!-- Meta Description -->
		<meta name="description" content="Haohan Wang, Trustworthy Machine Learning, UIUC, CMU, Computational Biology, Eric Xing">
		<!-- Meta Keyword -->
		<meta name="keywords" content="Haohan Wang, Trustworthy Machine Learning, UIUC, CMU, Computational Biology, Eric Xing">
		<!-- meta character set -->
		<meta charset="UTF-8">
		<!-- Site Title -->
		<title>Haohan Wang's Publication</title>
		<link rel="icon" href="favicon.ico" type="image/x-icon">

		<link href="https://fonts.googleapis.com/css?family=Poppins:100,200,400,300,500,600,700" rel="stylesheet"> 
			<!--
			CSS
			============================================= -->
			<link rel="stylesheet" href="css/linearicons.css">
			<link rel="stylesheet" href="css/font-awesome.min.css">
			<link rel="stylesheet" href="css/bootstrap.css">
			<link rel="stylesheet" href="css/magnific-popup.css">
			<link rel="stylesheet" href="css/jquery-ui.css">				
			<link rel="stylesheet" href="css/nice-select.css">							
			<link rel="stylesheet" href="css/animate.min.css">
			<link rel="stylesheet" href="css/owl.carousel.css">				
			<link rel="stylesheet" href="css/main.css">
		</head>
		<body>	
		  <header id="header">
		    <div class="container main-menu">
		    	<div class="row align-items-center justify-content-between d-flex">
			      <div id="logo">
			        <a href="index.html"></a>
			      </div>
			      <nav id="nav-menu-container">
			        <ul class="nav-menu">
			          <li><a href="index.html">Home</a></li>
			          <li><a href="index.html#research">Research</a></li>
			          <li><a href="index.html#news">Recent Updates</a></li>
			          <li><a href="index.html#open">Collaborations</a></li>
			          <li><a href="index.html#talks">Talks</a></li>
			          <li><a href="publications.html">Publications</a></li>
			          <li><a href="index.html#about">About</a></li>
			        </ul>
			      </nav><!-- #nav-menu-container -->		    		
		    	</div>
		    </div>
		  </header><!-- #header -->

					
		<nav id="fh5co-main-nav" role="navigation">
			<a href="#" class="js-fh5co-nav-toggle fh5co-nav-toggle active"><i></i></a>
			<div class="js-fullheight fh5co-table">
				<div class="fh5co-table-cell js-fullheight">
					<ul>
					<li><a href="index.html">Home</a></li>
					<li><a href="publications.html">Publications</a></li>
					<li><a href="index.html">Contact</a></li>
					</ul>
				</div>
			</div>
		</nav>
		
		
		<div id="fh5co-page">
			<div id="fh5co-portfolio-section">
				<section class="banner-area">
				<div class="container">
					<div class="row">
						<div class="col-md-12 col-md-offset-1 text-center heading-section">
							<p>  </p>
							<h1>Publications</h1>
							<p>  </p>
						</div>
						<p>  </p>
						<div class="col-md-12 col-md-offset-1 text-center heading-section"><h6 align="left">Year:
							[<a href="#2023">2023</a>]
							[<a href="#2022">2022</a>]
							[<a href="#2021">2021</a>] 
							[<a href="#2020">2020</a>] 
							[<a href="#2019">2019</a>] 
							[<a href="#2018">2018</a>] 
							[<a href="#2017">2017</a>] 
							[<a href="#2016">2016</a>]  
							[<a href="#pre">Previous</a>]</h6>
						<p>  </p>
						<p>  </p>
						<h6 align="left">Topic:
							[<a href="javascript:sDiv('tag');">All</a>] 
							[<a href="javascript:sDiv('tagCB');">Computational Biology</a>]
							[<a href="javascript:sDiv('tagML');">ML & DL</a>]  
							[<a href="javascript:sDiv('tagStat');">Statistics</a>]  </h6>
						<p>  </p>
						<p align="left"><a href="https://scholar.google.com/citations?user=nZxJGeUAAAAJ&hl=en" target="_blank">[google scholar page]</a></p>
						<p>  </p>
						<br>
						</div>
					</div>
				</div>
				</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
<a name="2023"></a>
<p> </p>
<h5 align="center">---2023---</h5>
<p> </p>
	<ul align="left">
<div id="tagML_Bao-ICML23">
  <li>
    Bao, W., <b>Wang, H.</b>, Wu, J., & He, J.
    <span style="font-weight: bold;">
      <a href="https://arxiv.org/abs/2306.06508" target="_blank">Optimizing the Collaboration Structure in Cross-Silo Federated Learning</a>
    </span>,
    in proceedings of 2023 International Conference on Machine Learning.
    <i>ICML 2023</i>
  </li>
</div>
<div id="tagML_Zhang-WSDM23">
  <li>
    Zhang, P., Guo, J., Li, C., Xie, Y., Kim, J. B., Zhang, Y., Xie, X., <b>Wamg, H.</b>, & Kim, S.
    <span style="font-weight: bold;">
      <a href="https://dl.acm.org/doi/abs/10.1145/3539597.3570445" target="_blank">Efficiently leveraging multi-level user intent for session-based recommendation via atten-mixer network</a>
    </span>,
    In Proceedings of the Sixteenth ACM International Conference on Web Search and Data Mining,
    <i>WSDM 2023</i>, pp. 168-176. (<font style="color:Brown">Best Paper Honorable Mention</font>)
  </li>
</div>
<div id="tagML_Xiang-AAAI23">
  <li>
    Xiang, K., Zhang, X., She, J., Liu, J., <b>Wang, H.</b>, Deng, S., & Jiang, S.
    <span style="font-weight: bold;">
      <a href="https://ojs.aaai.org/index.php/AAAI/article/view/25395" target="_blank">Toward Robust Diagnosis: A Contour Attention Preserving Adversarial Defense for COVID-19 Detection</a>
    </span>,
    In Proceedings of the AAAI Conference on Artificial Intelligence, vol. 37, no. 3, pp. 2928-2937.
    <i>AAAI 2023</i>
  </li>
</div>
<div id="tagML_Zhang-AAAI23"><li>Zhang Y.* and <b>Wang H.</b> <span style="font-weight: bold;"><a href="https://openreview.net/forum?id=UQY0bqcl_mX" target="_blank">Photong: Generating 16-Bar Melodies from Images</a></span>,
AAAI 2023 Workshop creativeAI homepage<i>
2023 
</i> (<font style="color:Orange">* K-12 Author</font>)
</li></div>
</ul>
			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
<a name="2022"></a>
<p> </p>
<h5 align="center">---2022---</h5>
<p> </p>
	<ul align="left">
<div id="tagML_tagStat_Wang-thesis22"><li><b>Wang H.</b>,<span style="font-weight: bold;"><a href="https://github.com/HaohanWang/HaohanWang.github.io/blob/master/file/haohanw_phd_lti_2021.pdf" target="_blank">Toward Robust Machine Learning by Countering Superficial Features</a></span>,
Phd Thesis<i>
2022 
</i>
</li></div>
<div id="tagCB_tagStat_Wang-JCB22"><li><b>Wang H.</b>, Lopez O, Xing E. P., and Wu W.<span style="font-weight: bold;"><a href="https://www.liebertpub.com/doi/full/10.1089/cmb.2022.0280" target="_blank">Kernel Mixed Model for Transcriptome Association Study</a></span>,
Journal of Computational Biology<i>
2022 
</i>
</li></div>
<div id="tagML_Wang-KDD22"><li><b>Wang, H.</b>, Huang Z., Wu X., Xing E.P. <span style="font-weight: bold;"><a href="https://arxiv.org/abs/2206.01909" target="_blank">Toward Learning Robust and Invariant Representations with Alignment Regularization and Data Augmentation</a></span>,
Proceedings of the ACM SIGKDD International Conference on Knowledge Discovery and Data Mining<i>
KDD 2022 
</i>
</li></div>
<div id="tagML_Wang-UAI22"><li><b>Wang, H.</b>, Huang Z., Zhang H., Lee Y.J., Xing E.P. <span style="font-weight: bold;"><a href="https://arxiv.org/abs/2111.03740" target="_blank">Toward Learning Human-aligned Cross-domain Robust Models by Countering Misaligned Features</a></span>,
Proceedings of the Uncertainty in Artificial Intelligence<i>
UAI 2022 
</i>
</li></div>
<div id="tagML_Wang-NAACL22"><li>Wang X, <b>Wang, H.</b>, Yang D. <span style="font-weight: bold;"><a href="https://arxiv.org/abs/2112.08313" target="_blank">Measure and Improve Robustness in NLP Models: A Survey</a></span>,
Proceedings of the Annual Conference of the North American Chapter of the Association for Computational Linguistics<i>
NAACL 2022 
</i>
</li></div>
<div id="tagML_Wang-CVPR22"><li><b>Wang, H.*</b>, Huang, Z.*, Huang, D., Lee, Y. J., & Xing, E. P.<span style="font-weight: bold;"><a href="https://openaccess.thecvf.com/content/CVPR2022/html/Huang_The_Two_Dimensions_of_Worst-Case_Training_and_Their_Integrated_Effect_CVPR_2022_paper.html" target="_blank">The Two Dimensions of Worst-Case Training and Their Integrated Effect for Out-of-Domain Generalization</a></span>,
Proceedings of the 2022 Annual Conference of the North American Chapter of the Association for Computational Linguistics<i>
CVPR 2022 
</i>
</li></div>
<div id="tagCB_tagStat_Miranda-JPM22"><li>Miranda, O., Fan P., Qi X., Yu Z., Ying J., <b>Wang H.</b>, Brent D.A., Silverstein J.C., Chen Y., Wang L.<span style="font-weight: bold;"><a href="https://www.mdpi.com/2075-4426/12/4/524" target="_blank">DeepBiomarker: identifying important lab tests from electronic medical records for the prediction of suicide-related events among PTSD patients</a></span>,
Journal of personalized medicine, 12(4), 524.
</li></div>
<div id="tagCB_tagStat_Wang-RECOMB22"><li><b>Wang H.</b>, Lopez O, Wu W., and Xing E. P.,<span style="font-weight: bold;"><a href="https://link.springer.com/chapter/10.1007/978-3-031-04749-7_7" target="_blank">A Network-structured High-dimension Variable Selection Method with P-values, for Gene Set Prioritization with Transcriptome Association Study Guided by Regulatory Network</a></span>,
International Conference on Research in Computational Molecular Biology<i>
RECOMB 2022 
</i>
</li></div>
</ul>
			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">							
<a name="2021"></a>
<p> </p>
<h5 align="center">---2021---</h5>
<p> </p>
	<ul align="left">
<div id="tagCB_tagStat_Wang-JCB21"><li><b>Wang H.</b>, Aragam B., and Xing E. P.,<span style="font-weight: bold;"><a href="https://arxiv.org/abs/2111.03739" target="_blank">Tradeoffs of Linear Mixed Models in Genome-wide Association Studies</a></span>,<i>
Journal of Computational Biology 29.3 (2022): 233-242
</i>
</li></div>
<div id="tagML_Ge-NIPS21"><li>Songwei Ge, Mishra S., <b>Wang H</b>, Li, C., and Jacobs. D., <span style="font-weight: bold;"><a href="https://proceedings.neurips.cc/paper/2021/hash/e5afb0f2dbc6d39b312d7406054cb4c6-Abstract.html" target="_blank">Robust Contrastive Learning Using Negative Samples with Diminished Semantics</a></span>,<i>
Advances in Neural Information Processing Systems
</i>(NeurIPS 2021). 
</li></div>
<div id="tagCB_tagStat_Wang-BMC21"><li><b>Wang H.</b>, Pei F., Vanyukov MM., Bahar I., Wu W., Xing E. P.,<span style="font-weight: bold;"><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03959-2" target="_blank">Coupled Mixed Model for Joint Genetic Analysis of Complex Disorders with Two Independently Collected Data Sets</a></span>,<i>
BMC bioinformatics 22.1 (2021): 1-14 
</i>
</li></div>
<div id="tagCB_Du-Bioinfo21"><li>Du X., <b>Wang H.</b>, Zhu Z., Zeng X., Chang YW., Zhang J., and Xu M.<span style="font-weight: bold;"><a href="https://academic.oup.com/bioinformatics/article-abstract/37/16/2340/6147019" target="_blank">Active Learning to Classify Macromolecular Structures in Situ for Less Supervision in Cryo-electron Tomography</a></span>,<i>
Bioinformatics (2021) 
</i>
</li></div>
</ul>	

			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
						
<a name="2020"></a>
<p> </p>
<h5 align="center">---2020---</h5>
<p> </p>
      <ul align="left">
<div id="tagML_tagCB_Zheng-plos20"><li>Zheng Y., <b>Wang H.</b>, Yang Z., Gao X., Xing E. P.,and Xu M, <span style="font-weight: bold;"><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008297" target="_blank">Poly(A)-DG: a Deep-learning-based Domain Generalization Method to Identify Cross-species Poly(A) Signal without Prior Knowledge from Target Species</a></span>,<i>
PLOS Computational Biology, 16(11): e1008297 
</i>
</li></div>	      	      
<div id="tagML_Huang-eccv20"><li><b>Wang H</b>, Huang Z., Xing E. P.,and Huang D, and  <span style="font-weight: bold;"><a href="https://link.springer.com/chapter/10.1007/978-3-030-58536-5_8" target="_blank">Self-Challenging Improves Cross-Domain Generalization</a></span>,<i>
Proceedings of European Conference on Computer Vision, (ECCV 2020)
</i>(ECCV 2020). (<font style="color:Orange">top 2% Oral</font>)
</li></div>
<div id="tagML_Wang-cvpr20"><li><b>Wang H</b>, Wu X., Huang Z., and Xing E. P., <span style="font-weight: bold;"><a href="http://openaccess.thecvf.com/content_CVPR_2020/html/Wang_High-Frequency_Component_Helps_Explain_the_Generalization_of_Convolutional_Neural_Networks_CVPR_2020_paper.html" target="_blank">High Frequency Component Helps Explain the Generalization of Convolutional Neural Networks</a></span>,<i>
Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition
</i>(CVPR 2020). (<font style="color:Orange">top 2% Oral</font>)
</li></div>
<div
 id="tagCB_tagML_Ge-RECOMB20"><li> Ge, S., <b>Wang, H.</b>., Alavi, A., Xing, E. and Bar-Joseph, Z, <span style="font-weight: bold;"><a
 href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2020.0439" target="_blank">Supervised Adversarial Alignment of scRNA-seq Data</a></span>,<i>
Proceedings of 24th International Conference on Research in Computational Molecular Biology
</i>(RECOMB 2020).
 </i></li></div>
<div
 id="tagCB_Wang-BMC_MG20"><li> <b>Wang, H.</b>, Vanyukov, MM., Xing, EP. & Wu, W , <span style="font-weight: bold;"><a
 href="https://link.springer.com/article/10.1186/s12920-020-0667-4" target="_blank">Discovering Weaker Genetic Associations Guided by Known Associations</a></span>,<i>
BMC medical genomics, 13(3), 1-10.
 </i></li></div>
<div
 id="tagCB_Wang-BMC_Bioinfo20"><li> <b>Wang, H.</b>, Yue, T., Yang, J., Wu, W., & Xing, EP., <span style="font-weight: bold;"><a
 href="https://link.springer.com/article/10.1186/s12859-019-3300-9" target="_blank">Deep Mixed Model for Marginal Epistasis Detection and Population Stratification Correction in Genome-wide Association Studies</a></span>,<i>
BMC bioinformatics, 20(23), 1-11..
 </i></li></div>
</ul>
			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
<a name="2019"></a>
<p> </p>
<h5 align="center">---2019---</h5>
<p> </p>
      <ul align="left">
<div id="tagML_Wang-nips19"><li><b>Wang H</b>, Ge S., Lipton Z. C. , and Xing E. P., <span style="font-weight: bold;"><a href="https://proceedings.neurips.cc/paper/2019/hash/3eefceb8087e964f89c2d59e8a249915-Abstract.html" target="_blank">Learning Robust Global Representations by Penalizing Local Predictive Power</a></span>,<i>
Proceedings of Thirty-fourth Conference on Neural Information Processing Systems
</i>(NeurIPS 2019).
</li></div>
<div id="tagML_Wang-iclr19"><li><b>Wang H</b>, He Z., Lipton Z. C. , and Xing E. P., <span style="font-weight: bold;"><a href="https://arxiv.org/abs/1903.06256" target="_blank">Learning Robust Representations by Projecting Superficial Statistics Out</a></span>,<i>
Proceedings of Seventh International Conference on Learning Representations
</i>(ICLR 2019). (<font style="color:Orange">top 1% Oral</font>)
</li></div>
<div
 id="tagML_Wang-aaai19"><li><b>Wang H</b>, Sun D., and Xing, E. P. <span style="font-weight: bold;"><a
 href="https://ojs.aaai.org/index.php/AAAI/article/view/4696" target="_blank">What If We Simply Swap the Two Text Fragments? A Straightforward yet Effective Way to Test the Robustness of Methods to Confounding Signals in Nature Language Inference Tasks</a></span>,<i>
Proceedings of Thirty-third AAAI Conference on Artificial Intelligence
 </i>(AAAI 2019). (Oral)
	  </li></div>
<div
 id="tagML_tagCB_Wang-psb19a"><li><b>Wang H</b>, Wu Z., and Xing E. P., <span style="font-weight: bold;"><a
 href="https://www.worldscientific.com/doi/abs/10.1142/9789813279827_0006" target="_blank">Removing Confounding Factors Associated Weights in Deep Neural Networks Improves the Prediction Accuracy for Healthcare Applications</a></span>,<i>
Proceedings of 24th Pacific Symposium on Biocomputing
 </i>(PSB 2019). (Oral)</li></div>
<div
 id="tagML_tagCB_Wang-psb19b"><li><b>Wang H</b>, Liu X., Tao Y., Ye W., Jin Q., Cohen W. W., and Xing E. P., <span style="font-weight: bold;"><a
 href="https://www.worldscientific.com/doi/abs/10.1142/9789813279827_0011" target="_blank">Automatic Human-like Mining and Constructing Reliable Genetic Association Database with Deep Reinforcement Learning</a></span>,<i>
Proceedings of 24th Pacific Symposium on Biocomputing
 </i>(PSB 2019).</li></div>
<div
 id="tagML_Xiao-nd19"><li>Xiao Y, Chen D, Wei S, Li Q, <b>Wang H</b>, and Xu M. <span style="font-weight: bold;"><a
 href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=nZxJGeUAAAAJ&cstart=20&pagesize=80&sortby=pubdate&citation_for_view=nZxJGeUAAAAJ:PELIpwtuRlgC" target="_blank">Rumor Propagation Dynamic Model Based on Evolutionary Game and Anti-rumor</a></span>,<i>
Nonlinear Dynamics. 2019 Jan 1;95(1):523-39.
 </i> </li></div>
</ul>
			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
<a name="2018"></a>
<p> </p>
<h5 align="center">---2018---</h5>
<p> </p>
      <ul align="left">
<div
 id="tagML_tagStat_tagCB_Wang-bioinfo18"><li><b>Wang H</b>, Lengerich B. J., Aragam B., and Xing E. P., <span style="font-weight: bold;"><a
 href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=nZxJGeUAAAAJ&cstart=20&pagesize=80&sortby=pubdate&citation_for_view=nZxJGeUAAAAJ:-f6ydRqryjwC" target="_blank">Precision Lasso: Accounting for Correlations and Linear Dependencies in High-Dimensional Genomic Data</a></span>,<i> Bioinformatics, PMID: 30184048 DOI:10.1093/bioinformatics/bty750 , 2018.
 </i></li></div>
<div
 id="tagCB_Wang-methods18a"><li><b>Wang H</b>, Liu X., Xiao Y., Xu M. and Xing E. P., <span style="font-weight: bold;"><a
 href="https://ieeexplore.ieee.org/abstract/document/8217649/" target="_blank">Multiplex Confounding Factor Correction for Genomic Association Mapping with Squared Sparse Linear Mixed Model</a></span>,<i>
Methods, 2018 Aug 1; 145: 33?40.
 </i>
<!--<b>(IEEE BIBM 2017 student award)</b>-->
</li></div>
<div
 id="tagStat_tagCB_Wang-methods18b"><li><b>Wang H</b>, Aragam B. and Xing E. P., <span style="font-weight: bold;"><a
 href="https://ieeexplore.ieee.org/abstract/document/8217687/" target="_blank">Variable Selection in Heterogeneous Datasets: A Truncated-rank Sparse Linear Mixed Model with Applications to Genome-wide Association Studies</a></span>,<i>
Methods, 2018 2018 Aug 1; 145: 2?9.
 </i></li></div>
<div
 id="tagML_tagCB_Yue-springer18"><li>Yue, T., & <b>Wang H</b>, <span style="font-weight: bold;"><a
 href="https://arxiv.org/abs/1802.00810" target="_blank">Deep Learning for Genomics: A Concise Overview</a></span>,<i>
arXiv preprint arXiv:1802.00810.
 </i></li></div>
</ul>
			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
<a name="2017"></a>
<p> </p>
<h5 align="center">---2017---</h5>
<p> </p>
      <ul align="left">
<div
 id="tagCB_Lee-methods17"><li>Lee, S., <b>Wang, H.</b>, & Xing, EP.</b>, <span style="font-weight: bold;"><a
 href="https://www.sciencedirect.com/science/article/pii/S1046202317300816" target="_blank">Backward Genome-Transcriptome-phenome Association Mapping</a></span>,<i>
Methods, 129, 18-23.
 </i></li></div>
<div
 id="tagML_Wang-arXiv17"><li><b>Wang, H.</b> & Raj, B., <span style="font-weight: bold;"><a
 href="https://arxiv.org/abs/1702.07800" target="_blank">On the Origin of Deep Learning</a></span>,<i>
arXiv preprint arXiv:1702.07800.
 </i></li></div>
<div
 id="tagML_Wang-icme17"><li><b>Wang, H.</b>, Meghawat, A., Morency, L. P., & Xing, E. P., <span style="font-weight: bold;"><a
 href="https://ieeexplore.ieee.org/abstract/document/8019301/" target="_blank">Select-Additive Learning: Improving Generalization Multimodal Sentiment Analysis</a></span>,<i>
In 2018 IEEE International Conference on Multimedia and Expo
 </i>(ICME 2017). <b><a href="http://www.cs.cmu.edu/~haohanw/SAL.html">[Data]</a><a href="https://github.com/HaohanWang/SelectAdditiveLearning">[Code]</a></b></li></div> 
</ul>
			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
<a name="2016"></a>
<p> </p>
<h5 align="center">---2016---</h5>
<p> </p>
      <ul align="left">					
<div
 id="tagML_tagCB_Yang-nipsws16"><li>Yang, J., <b>Wang, H.</b>, Zhu, J., & Xing, E. P., <span style="font-weight: bold;"><a
 href="http://www.cs.cmu.edu/~haohanw/papers/2016/Yang_etal_nipsws16.pdf" target="_blank">SeDMiD for Confusion Detection: Mind State from Time Series Brain Wave Data</a></span>,<i>
In TSW workshop NIPS 2016
 </i></li></div> 								
<div
 id="tagML_tagCB_Wang-bibm16"><li><b>Wang H</b>, Yang, J., <span style="font-weight: bold;"><a
 href="https://arxiv.org/abs/1611.10252" target="_blank">Multiple Confounders Correction with Regularized Linear Mixed Effect Models, with Application in Biological  Processes.</a></span>,<i>in 2016 IEEE International Conference on Bioinformatics and Biomedicine.
 </i>(BIBM 16)</li></div>
</ul>
			</div>
		</div>
	</div>	
</section>
<section class="services-area">
	<div class="container">
		<div class="row">
			<div class="col-md-12 col-md-offset-1 text-center heading-section">
<a name="pre"></a>
<p> </p>
<h5 align="center">---Previous---</h5>
<p> </p>
      <ul align="left">										
<div
 id="tagML_tagCB_Gupta-bibm15"><li>Gupta, A., <b>Wang, H.</b>, & Ganapathiraju, M., <span style="font-weight: bold;"><a
 href="https://ieeexplore.ieee.org/abstract/document/7359871/" target="_blank">Learning Structure in Gene Expression Data Using Deep Architectures, an Application to Gene Clustering.</a></span>,<i>in 2015 IEEE International Conference on Bioinformatics and Biomedicine.
 </i>(BIBM 15)</li></div>
<div
 id="tagML_Moon-nipsws15"><li>Moon, S., Kim, S., & <b>Wang, H.</b>, <span style="font-weight: bold;"><a
 href="https://www.researchgate.net/profile/Haohan-Wang-4/publication/269417514_Multimodal_Transfer_Deep_Learning_for_Audio_Visual_Recognition/links/566c86cc08aea0892c4fd29e/Multimodal-Transfer-Deep-Learning-for-Audio-Visual-Recognition.pdf" target="_blank">Multimodal Transfer Deep Learning for Audio Visual Recognition</a></span>,<i>
MMML Workshop NIPS 2015
 </i></li></div>
<div
 id="tag_Wang-dev13"><li><b>Wang, H.</b>, Raza, A. A., Lin, Y., & Rosenfeld, R., <span style="font-weight: bold;"><a
 href="https://dl.acm.org/doi/abs/10.1145/2537052.2537062" target="_blank">Behavior Analysis of Low-literate Users of a Viral Speech-based Telephone Service.</a></span>,<i>In Proceedings of the 4th Annual Symposium on Computing for Development.
 </i>(ACM DEV 13)</li></div>
<div
 id="tagML_tagCB_Wang-aied13"><li><b>Wang, H.</b>, Li, Y., Hu, X., Yang, Y., Meng, Z., & Chang, K. M., <span style="font-weight: bold;"><a
 href="https://people.csail.mit.edu/zp/moocshop2013/paper_8.pdf" target="_blank">Using EEG to Improve Massive Open Online Courses Feedback Interaction</a></span>,<i>
In AIED Workshops.</i>
<b><a href="https://www.kaggle.com/wanghaohan/eeg-brain-wave-for-confusion" target="_blank">[Data]</a><a href="https://github.com/HaohanWang/BioImaging" target="_blank">[Code]</a><a href="https://www.youtube.com/watch?v=ACjLuMvbAnM" target="_blank">[Video]</a></b>
</li></div>
</ul>
			</div>
		</div>
	</div>
</section>
</div>
</div>	

            <!-- start footer Area -->
            <footer class="footer-area section-gap">
                <div class="container">
                    <div class="row">
                    </div>
                </div>
            </footer>
            <!-- End footer Area -->		

			<script src="js/vendor/jquery-2.2.4.min.js"></script>
			<script src="js/popper.min.js"></script>
			<script src="js/vendor/bootstrap.min.js"></script>			
			<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBhOdIF3Y9382fqJYt5I_sswSrEw5eihAA"></script>			
  			<script src="js/easing.min.js"></script>			
			<script src="js/hoverIntent.js"></script>
			<script src="js/superfish.min.js"></script>	
			<script src="js/jquery.ajaxchimp.min.js"></script>
			<script src="js/jquery.magnific-popup.min.js"></script>	
    		<script src="js/jquery.tabs.min.js"></script>						
			<script src="js/jquery.nice-select.min.js"></script>	
            <script src="js/isotope.pkgd.min.js"></script>			
			<script src="js/waypoints.min.js"></script>
			<script src="js/jquery.counterup.min.js"></script>
			<script src="js/simple-skillbar.js"></script>							
			<script src="js/owl.carousel.min.js"></script>							
			<script src="js/mail-script.js"></script>	
			<script src="js/main.js"></script>	
		</body>
	</html>
